adafruit_button.py

```diff
@@ -59,12 +65,68 @@ class Button(displayio.Group):
     :param selected_outline: Inverts the outline color.
     :param selected_label: Inverts the label color.
     """
+    RECT: int = const(0)
+    ROUNDRECT: int = const(1)
+    SHADOWRECT: int = const(2)
+    SHADOWROUNDRECT: int = const(3)
+
+    def __init__(
+        self,
+        *,
+        x: int,
+        y: int,
+        width: int,
+        height: int,
+        name: Optional[str] = None,
+        style: int = RECT,
+        fill_color: Optional[int] = 0xFFFFFF,
+        outline_color: Optional[int] = 0x0,
+        label: Optional[Label] = None,
```

Comment:
`label` is expecting a `string` type. It will eventually put that string into a Label object, but when passed in here it should be a `string` I believe.

Reply:
Hi, so here is where I was struggling during Sprint. When `label` is not Label object, both property and property setter for it throw errors, as we utilize Label object's arguments (`.text`, `.bounding_box` etc.) on variable we hinted as `string`. 

Is there any way to work-around this issue, so mypy does not throw errors, when checking?

Thanks!

Reply:
@perkin-borowa  I think the type hinter may be confused by this line in the init function:
```
self._label = label
```
This is setting the argument `label` onto `self._label` which is technically mixing types. The rest of the code is written to assume that `self._label` will be a Label object. 

It never caused any other trouble because the last line of the init function ends up superceding this one that is erroneously setting `self._label` to the string argument.
```
self.label = label
```
That line calls the setter which is expecting the text to get passed into it. 

Try removing 
```
self._label = label
```
And see if mypy still reports any errors about Label object vs. string for the argument.